payments/payments.go

```diff
@@ -373,6 +379,17 @@ type (
 		Payer        Payer          `json:"payer,omitempty"`
 		Installments *Installments  `json:"installments,omitempty"`
 	}
+
+	Product struct {
```

Comment:
We already have a Product struct in `payments/nas/payments.go` we should try to reuse that struct